index.html

```diff
@@ -1 +1,31 @@
-<!--beginning of html-->
\ No newline at end of file
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+    <meta name="HandheldFriendly" content="true">
+    <meta name="keywords" lang="en-us" content="events,plot,night,life">
+    <title>Events World</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+</head>
+<body>
+    <div id="navigation_bar">
+        <span><h1 id="navigation_title">Events World</h1></span>
+    </div>
+<img id ="landing_image"src="pic.jpg" width="50%">
+    <div class="registration_form">
+        <form action="" method="POST">
+            <span>First name:</span>
+            
+            <p align="center"> <input id="fields" type="text" name="name" placeholder="Enter your first name"></p>
```

Comment:
The align attribute is not supported in HTML5. It;s better to use text-align instead.